websockets/kiturawebsockets.html

```diff
@@ -0,0 +1,315 @@
+          <li class="nested-sidebar-item"><a href="../deploying/kubernetes.html">Kubernetes</a></li>
+          <li class="nested-sidebar-item"><a href="../deploying/cloud-foundry.html">Cloud Foundry</a></li>
+        </ul>
+        <li class="sidebar-item collapsible">Configuring</li>
+        <ul class="nested-sidebar-list content">
+          <li class="nested-sidebar-item"><a href="../configuring/enabling-nio.html">Enabling SwiftNIO</a></li>
+        </ul>
+      </ul>
+    </aside>
+    <div id="burgerIcon" class="burger-icon" onclick="showSidebar()">
+      <div class="burger-line"></div>
+      <div class="burger-line"></div>
+      <div class="burger-line"></div>
+    </div>
+    <div class="docs-item-2 search-container">
+    </div>
+    <nav class="docs-item-3 docs-nav">
+      <button id="api-button" class="apiref-button" type="button" name="button" onclick="window.open('https://ibm-swift.github.io/Kitura/')">API Reference</button>
+      <a class="nav-item" target="_blank" href="http://slack.kitura.io/">Need help?</a>
+      <a class="nav-item" target="_blank" href="https://github.com/IBM-Swift/kitura.io/issues">Found an issue?</a>
+    </nav>
+    <div id="doc-container" class="docs-item-4 docs-window">
+      <main>
+        <h1 class="heading-1">Kitura-WebSocket Echo Server</h1>
+        <p class="block-text">The application we are going to create is a very simple Echo Server, this is a server that will receive a message from the client and then echo back the same message to the client, this is the simplest implementation of WebSockets.</p>
+        <h2 class="heading-2"><span class="blue-text">Step 1:</span> Pre-requisites</h2>
+        <p class="block-text">Before creating the server, we need to make sure everything is set up correctly, <a href="https://nodejs.org/en/" target="_blank">Node.js</a> must be installed and the project directories should be created with this terminal command:</p>
+        <pre><code>mkdir EchoServer
+cd EchoServer
+touch Package.swift
+mkdir Sources
+cd Sources
+mkdir ChatServer
+cd ChatServer
+touch ChatService.swift
+touch main.swift</code></pre>
+      <h2 class="heading-2"><span class="blue-text">Step 2:</span> Setting up the Server</h2>
+      <p class="block-text">Add the following content to your <strong>Package.swift</strong> file:</p>
+      <pre><code class="language-swift">// swift-tools-version:4.0
+import PackageDescription
+
+let package = Package(
+    name: "ChatServer",
+    dependencies: [
+         .package(url: "https://github.com/IBM-Swift/Kitura.git", .upToNextMinor(from: "2.7.0")),
+         .package(url: "https://github.com/IBM-Swift/HeliumLogger.git", from: "1.0.0"),
+         .package(url: "https://github.com/IBM-Swift/Kitura-WebSocket.git", from: "2.1.2")
+    ],
+    targets: [
+        .target(
+            name: "ChatServer",
+            dependencies: ["Kitura", "HeliumLogger", "Kitura-WebSocket"]),
+    ]
+)</code></pre>
```

Comment:
// Use alternate implementation of Kitura-WebSocket while building in NIO mode
this comment is not lined up properly in the code.

You should also link out to why we have this code snippet included to the Enabling NIO section from your info box.